### PR TITLE
Add Execution Environment detail page with tabs and actions

### PR DESCRIPTION
**Pull request by jerabekjiri**
Screenshot from 2024-01-29 15-58-17

### CODE REVIEW

File: frontend/hub/execution-environments/ExecutionEnvironmentPage/hooks/useExecutionEnvironmentPageActions.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        label: t('Edit'),
+        onClick: (ee) =>
+          pageNavigate(HubRoute.EditExecutionEnvironment, { params: { id: ee?.name } }),
+      },
+      { type: PageActionType.Seperator },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Single,
+        label: t('Use in controller'),
+        onClick: () => alert('WIP'),
```
Comment (himdel) [inline frontend/hub/execution-environments/ExecutionEnvironmentPage/hooks/useExecutionEnvironmentPageActions.tsx:35]: just make sure to remove all the `alert`s soon :)

**Review by himdel — APPROVED** [review]

File: frontend/hub/execution-environments/ExecutionEnvironmentPage/hooks/useExecutionEnvironmentPageActions.tsx (hunk 1 above)
Comment (jerabekjiri) [reply frontend/hub/execution-environments/ExecutionEnvironmentPage/hooks/useExecutionEnvironmentPageActions.tsx:35]: I'll remove that, wasn't paying attention

**Comment by jerabekjiri** [comment]
@himdel Is it okay to change EE paths to `execution-environments/alpine/activity`, the old UI is `containers/alpine/_content/activity/`?

**Comment by himdel** [comment]
yeah, UI paths will be different in the new UI, that's fine :)

(just make sure it still works when the container is `foo/bar` instead of `alpine`)

**Comment by jerabekjiri** [comment]
oh, that's a good point, I have to test what happens with `foo/bar` format, probably needs to be fixed.

**Comment by jerabekjiri** [comment]
Looks like react router escapes into `foo%2Fbar`, not sure if that's the desired output

**Comment by himdel** [comment]
Yeah, as long as `foo%2Fbar` in the url works, and the rest of the ui works as expected, I think that's close enough for now :)

The old ui implements it as 2 sets of routes, one with `/:namespace/:name/`, other with just `/:name/` but it's a bit of a hack, it could be replaced with optional params now (in the old ui, along the lines of `/:namespace?/:name/`), but the hierarchical router here may need some work to support that...

We can leave supporting foo/bar in the url as a separate issue, as long as foo%2Fbar works :)

**Pull request merged**